examples/insert_data_formats_overview.ts

```diff
@@ -0,0 +1,162 @@
+import { createClient } from '@clickhouse/client'
+import type { DataFormat } from '@clickhouse/client-common'
+import {
+  type InputJSON,
+  type InputJSONObjectEachRow,
+} from '@clickhouse/client-common'
+
+/**
+ * An overview of available formats for inserting your data, mainly in different JSON formats.
+ * For "raw" formats, such as:
```

Comment:
Let's add a comment: The list might be incomplete, so we ask users to submit more examples. It's too easy to add a new format without updating the example.